pkg/k8sops/m3db/generators_test.go

```diff
@@ -37,8 +37,8 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 
 	crdutils "github.com/ant31/crd-validation/pkg"
-	"k8s.io/utils/pointer"
 	"github.com/stretchr/testify/assert"
+	"k8s.io/utils/pointer"
```

Comment:
Nit since it seems like the import order linter isn't working but can you move this to the other `k8s.io` import groups?